yeetd/main.swift

```diff
@@ -8,22 +8,28 @@
 import Foundation
 import OSLog
 
+UserDefaults.standard.setValue(true, forKey: "killapsd")
+
 // TODO: should probably just have these in a file somewhere
-// Configuration bits
-let processesToWatch: Set<String> = [
-    "AegirPoster",
-    "InfographPoster",
-    "CollectionsPoster",
-    "ExtragalacticPoster",
-    "KaleidoscopePoster",
-    "EmojiPosterExtension",
-    "AmbientPhotoFramePosterProvider",
-    "PhotosPosterProvider",
-    "AvatarPosterExtension",
-    "GradientPosterExtension",
-    "MonogramPosterExtension",
-    "apsd",
-]
+var processesToWatch: Set<String> {
+    var processes: Set<String> = [
+        "AegirPoster",
+        "InfographPoster",
+        "CollectionsPoster",
+        "ExtragalacticPoster",
+        "KaleidoscopePoster",
+        "EmojiPosterExtension",
+        "AmbientPhotoFramePosterProvider",
+        "PhotosPosterProvider",
+        "AvatarPosterExtension",
+        "GradientPosterExtension",
+        "MonogramPosterExtension"
+    ]
+    if UserDefaults.standard.bool(forKey: "killapsd") {
```

Comment:
It would be neat/nice to be able to insert this in the process check loop instead (e.g., a person could change this default at runtime and it would automatically work without restarting the daemon).

Reply:
I'm not sure what the cleanest way is to do this so I might simmer on it for a bit. I do however think we should have a loop check in some fashion.